src/tools/qt-gui/src/guibackend.cpp

```diff
@@ -108,11 +107,11 @@ void GUIBackend::serialise(TreeViewModel *model)
 
 		if (root == "system")
 		{
-			model->sink(model->model().at(0), keys, "system", k);
+			model->sink(model->model().at(0), keys, k);
 		}
 		else if (root == "user")
 		{
-			model->sink(model->model().at(1), keys, "user", k);
+			model->sink(model->model().at(1), keys, k);
 		}
```

Comment:
there are some more namespaces now (see doc/NAMESPACES.md). You can use

     elektraNamespace keyGetNamespace(Key const* key)

do make sure you won't miss one (compiler warns you if you haven't handeled all cases in switch)